internal/pkg/cli/options/options.go

```diff
@@ -41,12 +40,7 @@ func New() *Options {
 
 func (o *Options) Load(logger log.Logger, osEnvs *env.Map, fs filesystem.Fs, flags *pflag.FlagSet) error {
 	// Load ENVs from OS and files
-	envs, err := o.loadEnvFiles(logger, osEnvs, fs)
-	if err == nil {
-		o.envs = envs
-	} else {
-		logger.Debug(err.Error())
-	}
```

Comment:
Problem bol tu.
Ak sa `env` subory nepodarilo nacitat, tak sa to zalogovalo ako `debug`,.
Ale `o.envs` zostala `nil`, a spadlo to v kode dalej.

Reply:
Preto tato operacia uz nemoze skoncit chybou.